packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/AttachmentDemos.md

```diff
@@ -46,12 +46,50 @@ import PFIconLogoReverse from '../ChatbotHeader/PF-IconLogo-Reverse.svg';
 
 ### Attach via upload button in message bar
 
+This demo displays unique attachment features, including:
+1. [`<ChatbotContent>` and `<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) components that contain:
+- An initial [user `<Message>`](/patternfly-ai/chatbot/chatbot-messages/#user-messages) with an attachment.
+- The ability to preview or edit the attachment using the [`<PreviewAttachment>` and `<AttachmentEdit>` components.](patternfly-ai/chatbot/chatbot-messages/#messages-with-attachments)
+- A PatternFly [`<Alert>`](/components/alert) to provide success or error messages for attachments.
```

Comment:
Link is broken? Unsure if we should link to PF or if this is ok as-is.

Reply:
this will be the correct link once it's pulled into org